fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/ExpressionOptimization.java

```diff
@@ -58,6 +59,13 @@ public class ExpressionOptimization extends ExpressionRewrite {
                     BetweenToEqual.INSTANCE
             )
     );
+
+    public static final List<ExpressionRewriteRule> OPTIMIZE_REWRITE_RULES_OR_ADD_RANGE = ImmutableList.of(
```

Comment:
this name make pepole understand as "optimize rewrite rules" or "add range" 
it is better to use the rule name

Reply:
fix